### ISSUE TITLE
Getting 'cannot find ./lib/mime' error

### ISSUE THREAD

**Issue by andymikulski** [feross/render-media#4]
Recently updated to node v5.5.0 and ran into this issue (while using the `webtorrent` package):

```
ERROR in ./~/render-media/index.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./lib/mime in [path]/node_modules/render-media
 @ ./~/render-media/index.js 3:26-47
```

It appears that the mime `require` isn't checking for the `.json` extension by default. Switching the code to the explicit extension resolved the error for me - `require('./lib/mime.json')`

Is it possible this is an environment error? Or would you accept a PR with the `.json` addendum? :smiley:

**Comment by feross** [comment]
Thanks for opening this issue. Not sure how this was working before!

Released as 2.1.0.

**Comment by masterwok** [comment]
I was having difficulty getting webpack to resolve the mime.json file using the webpack config sample on the WebTorrent readme.

After removing the following from the webpack config it resolved the issue.

```
 {
        test: /\.json$/,
        loader: 'json'
 }
```
Not sure if this is specific to my project or if the WebTorrent readme needs to be updated...
